core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/RemoveMixin.java

```diff
@@ -0,0 +1,49 @@
+package de.valtech.aecu.core.groovy.console.bindings.actions.resource;
+
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import de.valtech.aecu.core.groovy.console.bindings.actions.util.MixinUtil;
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.nodetype.NoSuchNodeTypeException;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+
+/**
+ * Removes a mixin from a resource/node if the mixin is present.
+ *
+ * @author Eric Manzi and Bart Thierens
+ */
+public class RemoveMixin implements Action {
+
+    private final String mixinName;
+
+    public RemoveMixin(String mixinName) {
```

Comment:
please add javadoc here

Reply:
what exactly would you like to see added?  It already has javdoc on class level

Reply:
just realized that we also don't have javadoc for other action constructors. so yeah we can ignore it, doesn't provide lot of additional context.